src/renderer/api/endpoints/helm-releases.api.ts

```diff
@@ -112,8 +112,8 @@ export const helmReleasesApi = {
     return apiBase.del(path);
   },
 
-  getValues(name: string, namespace: string) {
-    const path = `${endpoint({ name, namespace })}/values`;
+  getValues(name: string, namespace: string, all: boolean) {
+    const path = `${endpoint({ name, namespace })}/values${all? "?all": ""}`;
```

Comment:
```suggestion
    const path = `${endpoint({ name, namespace })}/values${all ? "?all" : ""}`;
```

Reply:
Are the spaces matter? It's already passed linting.